smartsim/_core/control/manifest.py

```diff
@@ -95,8 +91,10 @@ def all_entity_lists(self) -> t.List[EntityList]:
         _all_entity_lists.extend(self.ensembles)
 
         db = self.db
+
```

Comment:
recommend changing the name of self.db if it is capable of representing a list of DBs and not just a single one now

Reply:
is dbs sufficient?